parts/windowsparams.t

```diff
@@ -23,4 +23,25 @@
         "description": "The source of the generalized blob which will be used to create a custom windows image for the agent virtual machines."
       },
       "type": "string"
+    },
+    "agentWindowsPublisher": {
+      "defaultValue": "MicrosoftWindowsServer",
+      "metadata": {
+        "description": "The publisher of windows image for the agent virtual machines."
+      },
+      "type": "string"
+    },
+    "agentWindowsOffer": {
+      "defaultValue": "WindowsServer",
+      "metadata": {
+        "description": "The offer of windows image for the agent virtual machines."
+      },
+      "type": "string"
+    },
+    "agentWindowsSku": {
+      "defaultValue": "2016-Datacenter-with-Containers",
```

Comment:
Is this default value shared by k8s?

Reply:
yes

Reply:
k8s uses a different default "agentWindowsSku": "Datacenter-Core-1709-with-Containers-smalldisk".